### PR TITLE
Services 903

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/GoogleConnectPage.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * Created by Ludwik on 2015-11-04.
+ */
+public class GoogleConnectPage extends WikiBasePageObject {
+
+  @FindBy(css = "#message")
+  private WebElement message;
+
+  @FindBy(css = ".abcRioButtonContentWrapper")
+  private WebElement SignUpButton;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/GoogleConnectPage.java]: MAJOR Rename this field "SignUpButton" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by PiotrGackowski** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/googleconnect/GoogleConnect.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.GoogleConnectPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+
+public class GoogleConnect extends NewTestTemplate {
+
+  @Test
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/googleconnect/GoogleConnect.java]: The test doesn't have any group

**Pull request merged**